readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -139,14 +140,12 @@ def append_conf(self):
         user_config = self.load_yaml_config()
 
         # Handle custom docs dirs
-        user_docs_dir = user_config.get('docs_dir')
```

Comment:
It seems like we were respecting the users `docs_dir` setting here, weren't we? This PR just changes it to default to `docs` for users who didn't define one?

Reply:
yeah, we still respect the value set by the user